dash_html_components/__init__.py

```diff
@@ -36,21 +36,21 @@
 
 _basepath = _os.path.dirname(__file__)
 _filepath = _os.path.abspath(_os.path.join(_basepath, 'package.json'))
-with open(_filepath) as f:
-    package = json.load(f)
+with open(_filepath) as _f:
+    _package = _json.load(_f)
 
-js_package_name = package['name']
-py_package_name = __name__
-js_version = package['version']
+_js_package_name = _package['name']
+_py_package_name = __name__
+_js_version = _package['version']
 
 _js_dist = [
     {
-        "relative_package_path": '{}.min.js'.format(py_package_name),
-        "dev_package_path": '{}.dev.js'.format(py_package_name),
+        "relative_package_path": '{}.min.js'.format(_py_package_name),
+        "dev_package_path": '{}.dev.js'.format(_py_package_name),
         "external_url": (
-            "https://unpkg.com/{}@{}/{}/{}.min.js"
-        ).format(js_package_name, js_version, py_package_name, py_package_name),
-        "namespace": py_package_name
```

Comment:
@Marc-Andre-Rivet to get [test_imports](https://github.com/plotly/dash-html-components/blob/89a6d6717801f1ff361a3b02a214079bf4bbd748/tests/test_dash_html_components.py#L6) to pass (and therefore to keep package contents clean). I guess these tests hadn't been run since `fix release, alpha 2` https://github.com/plotly/dash-html-components/commit/cf1256c0649729113ffc2a78ad4d2f9d18ceb58f?

Reply:
Oh my.. thanks!